src/models/course.ts

```diff
@@ -23,13 +26,14 @@ export const courseSchema = new dynamoose.Schema(
 );
 
 export interface CourseInterface extends Document {
-    id: string;
+    id: string
     name: string;
     desiredGrade: number | FractionInterface;
-    courseItems: CourseItemInterface[];
-    owner: string;
+    courseItems?: Set<CourseItemInterface>;
```

Comment:
I don't think we want this property optional. It should be `courseItems: Set<CourseItemInterface>` and we should just specify an empty set. This has come up a few times with the team so I might be wrong about this being possible, so correct me if I am wrong.

Reply:
I am wrong, let's keep this optional.